cypress/integration/landingPage.spec.ts

```diff
@@ -38,10 +38,6 @@ describe("Upload from the the landing page", () => {
   });
 
   it("should disable to continue if only file is provided", () => {
-    // we need a valid bcp47Tag to update the packageInfo
-    cy.data("autocomplete-label").scrollIntoView().should("be.visible");
```

Comment:
Do these lines cause problems with the newly refactored code?

Reply:
Yes. Although, the thing about refactoring is that, if you've done it right, it should cause no effect on the tests.

I'm honestly pretty confused on why this was working before. The issue here is that typing "zh" in the language input would open the autocomplete selection box, which would then obscure the drag-and-drop area, making the rest of the test fail because the file upload area is inaccessible. However, this test "it should disable to continue if only file is provided" should not even _touch_ the language input... so why... was it doing this stuff in the first place?